docs/_pages/how-to-guide.md

```diff
@@ -7,15 +7,28 @@ toc: true
 # Prerequisites
 * [terraform >= 0.12](https://www.terraform.io/)
   - credentials configured for an AWS account
+  - note, some modules are pinned to 0.14 It is advised to use this version for now for best compatability. 
 * [terragrunt >= v0.21](https://github.com/gruntwork-io/terragrunt)
+* [AWS CLI >= 2.0](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+  - used for logging in to AWS
 
 # How To Use
-You can use the following command to see the baseline infrastructure deployed to
-your account:
+Validate that you have all tools installed correctly. You can do this by typing `{tool} --version` into the comand line to validate the tool is in your PATH and the version is appropriate. Note that in some installation methods of terragrunt (namely, Brew) will attempt to install the latest version of Terraform with it. This can cause a dependency issue. You may have to manually uninstall undesired versions and add the correct ones to your PATH. 
+
+After all tools are validated, you must log into the AWS CLI to set your environment config. This generally involves creating programmatic access for a user account and setting it up with `aws configure`. [Instructions for configuring your CLI environment to the desired AWS account can be found here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
```

Comment:
Nit: The aws cli doesn't exactly have a "login" method. This is much different than Azure. The *only* login method of the aws cli is to login to AWS SSO. You *can* use the aws cli to setup a config that tools can use to authenticate to AWS, but past `aws configure` the aws cli is not in that loop at all. And it is not even necessary to use the aws cli to setup the config file.

Also, this could use some more nuance. We need to be careful making recommendations about how to authenticate to AWS. IAM Users, despite the name, are not for humans. All human access to AWS should be using some form of SSO. All machine access should be using execution roles, and only in extreme cases should a permanent access key ever be created.

Reply:
attempted to answer nitpick!